doc/whatsnew/3/3.1/index.rst

```diff
@@ -12,5 +12,111 @@
 Summary -- Release highlights
```

Comment:
I think the date just above this line is okay to update now, too? 🤔

Reply:
Hurg, right 🤦